test/integration/test_konnect_entities.go

```diff
@@ -0,0 +1,124 @@
+package integration
+
+import (
+	"testing"
+	"time"
+
+	"github.com/Kong/sdk-konnect-go/models/components"
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	testutils "github.com/kong/gateway-operator/pkg/utils/test"
+	"github.com/kong/gateway-operator/test"
+	"github.com/kong/gateway-operator/test/helpers"
+
+	configurationv1alpha1 "github.com/kong/kubernetes-configuration/api/configuration/v1alpha1"
+	konnectv1alpha1 "github.com/kong/kubernetes-configuration/api/konnect/v1alpha1"
+)
+
+func TestKonnectEntities(t *testing.T) {
+	ns, _ := helpers.SetupTestEnv(t, GetCtx(), GetEnv())
+
+	testID := uuid.NewString()[:8]
```

Comment:
Why the length of `8`? It's not obvious

Reply:
Added a comment (tldr: Konnect does not like too long names).